### PR TITLE
build src base

### PR DESCRIPTION
**Pull request by nhatanh19**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,44 +1,44 @@
...
-Open on web: http://localhost:1234
+### Practice_HTML_CSS
+
+# OVERVIEW
+
+ - This document introduces my project with time to do it, the target i want to get, the technical i have used, the link to view my website and source code of it and the final is the way to run my project on your local machine.
+
+## TIMELINE:
+
+-  3 Week (September 23 - October 6)
```
Comment (Hoa0304) [inline README.md]: Your timeline seems wrong

File: package.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "practice_html_css_3",
+  "version": "1.0.0",
+  "description": "- This document introduces my project with time to do it, the target i want to get, the technical i have used, the link to view my website and source code of it and the final is the way to run my project on your local machine.",
+  "main": "index.js",
```
Comment (Hoa0304) [inline package.json]: ?

File: README.md (hunk 1 above)
Comment (nhatanh19) [reply README.md]: Thanks for the reminder

File: package.json (hunk 2 above)
Comment (Hoa0304) [reply package.json]: I mean what effect does this line have?

**Pull request merged**